packages/core/src/model.js

```diff
@@ -2105,7 +2105,7 @@ ${associationOwner._getAssociationDebugList()}`);
         throw new Error(`${dialect} does not support the ignoreDuplicates option.`);
       }
 
-      if (options.updateOnDuplicate && !['mysql', 'mariadb', 'sqlite', 'postgres', 'ibmi'].includes(dialect)) {
+      if (options.updateOnDuplicate && !['mysql', 'mariadb', 'sqlite', 'postgres', 'ibmi', 'cockroachdb'].includes(dialect)) {
```

Comment:
This should become a new property in index.ts (the DialectSupports object) since it seems that less than half support this

Reply:
Could you shed a bit more context here? Should it be like a property defined inside the inserts object like this 
`bulkInsert: {
      ignoreDuplicates: string,
      updateOnDuplicate: boolean | string,
    },`

Reply:
I missed this mesasge. Yes, that is similar to what I had in mind